models/public/anti-spoof-mn3/accuracy-check.yml

```diff
@@ -0,0 +1,29 @@
+models:
+  - name: anti-spoofing-mn3
+
+    launchers:
+      - framework: dlsdk
+        adapter: classification
+        tags:
+          - FP32
+        model: anti-spoof-mn3.xml
+        weights: anti-spoof-mn3.bin
+        device: CPU
+        inputs:
+          - name: actual_input_1
+            type: INPUT
+            shape: 1,3,128,128
```

Comment:
it is not necessary if your model has only one input and its shape described in IR